### ISSUE TITLE
How To - Network Topology - draw.io issue

### ISSUE THREAD

**Issue by NoahParkerAHEAD** [microsoft/ARI#246]
When I upload the xml to draw.io I get this error message. 

Error
BPWXqFOopaEeQmAnsZIH-4: Duplicate ID

I found the duplicate ID in the xml, but I don't understand how to resolve it. 
It is found found twice in the xml:

<object label=" example-subnet-01 " id="BPWXqFOopaEeQmAnsZIH-4">
<mxCell style="rounded=0;whiteSpace=wrap;fontSize=16;html=1;sketch=0;fontFamily=Helvetica;" vertex="1" parent="eKjAprXSwLbYxMZtdhfV-1">
<mxGeometry x="20" y="40" width="200" height="200" as="geometry"/>

I want to say one is found in the route table

**Comment by Claudio-Merola** [comment]
Hi,

That should not be happening. I don't advice to solve these types of problems directly in the XML file. The logic of the xml file is a little complicated and editing it directly might be a nightmare. 

Have you tried to run the script again to generate a new xml file? do you get the same error?

**Comment by NoahParkerAHEAD** [comment]
I agree we shouldn't be editing the xml file. I believe that the resource IDs are getting double referenced due to route table subnet association. As well as subnets being reference in the Vnet as well for networking visualization. Have you used this solution in an account that has a route table in it?  I am curious if it worked.

**Comment by Claudio-Merola** [comment]
Yes, the solution was used in environments with UDRs and UDRs associates with different subnets

**Comment by containerpope** [comment]
@Claudio-Merola I have the same issue currently. When I changed some IDs it works again. In my case, the affected network components are linked with private endpoints. I added some numbers to make it work, because I had only 6 conflicts in total, but maybe that screenshot can help to resolve the issue.

![Image](https://github.com/user-attachments/assets/ae0391f6-1cc9-4ac0-8395-3616c5de2e72)
